.github/workflows/main.yml

```diff
@@ -18,10 +18,17 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: cargo test
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2021-07-26
+          components: rustfmt, clippy
```

Comment:
In order to get access to the Rustc API, we need to install a few components, as defined [in the docs](https://github.com/iomentum/cargo-breaking/blob/a381567ed4ab4ebc22d9be999977c759f314b2b9/book/src/1_foreword.md).

This is needed for each CI job unfortunately.